.github/workflows/unit_test_200gb_CI.yml

```diff
@@ -76,12 +76,42 @@ jobs:
 
       - name: Start Vespa
         run: |
+          retry_command() {
+            local n=1
+            local max=5
+            local delay=5
+            while true; do
+                "$@" && break || {
+                    if [[ $n -lt $max ]]; then
+                        ((n++))
+                        echo "Command failed. Attempt $n/$max:"
+                        sleep $delay;
+                    else
+                        echo "The command has failed after $n attempts."
+                        return 1
+                    fi
+                }
+            done
+          }
+          
+          echo "Updating package list"
+          retry_command apt-get update -y
```

Comment:
What are the causes of the apt command failures? why do we need to retry in the first place?

Reply:
Not sure, I only know it happens after `apt-get update`. Some sort of lock issue that happens. @farshidz can you confirm you also get this?

Reply:
`unattended-upgrades` process. Trying to uninstall it in another branch